pkg/loop/telem.go

```diff
@@ -66,9 +67,16 @@ func SetupTracing(config TracingConfig) error {
 	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
 	defer cancel()
 
+	var creds credentials.TransportCredentials
+	if config.TLSCertPath != "" {
+		creds, _ = credentials.NewClientTLSFromFile(config.TLSCertPath, "")
```

Comment:
Unhandled error?

Reply:
Yes, in this revision I chose to validate in `parse` in `pkg/loop/config.go`